hw/xbox/nv2a.c

```diff
@@ -1309,11 +1311,14 @@ typedef struct VertexAttribute {
     unsigned int converted_size;
     unsigned int converted_count;
 
+    float *inline_buffer_buffer;
```

Comment:
Done and also renamed gl_inline_buffer_buffer to be more similar to converted_buffer / gl_converted_buffer ;)